content/docs/user-guide/project-structure/pipelines-files.md

```diff
@@ -156,6 +160,49 @@ stages:
 DVC will track simple param values (numbers, strings, etc.) used in `${}` (they
 will be listed by `dvc params diff`).
 
+### Dict Unpacking
```

Comment:
This file was already too large but I think the extraction (https://github.com/iterative/dvc.org/issues/3670) should be addressed in a separate P.R.